src/dioptra/restapi/v1/jobs/service.py

```diff
@@ -1055,3 +1204,68 @@ def create(
             )
 
         return {"mlflow_run_id": job.mlflow_run.mlflow_run_id}
+
+
+class ExperimentMetricsService(object):
```

Comment:
I think this should go under `experiments/service.py`

Reply:
I put this here because it relies on both ExperimentJobService and JobIdMetricsService which causes a circular import if you put it in experiments/service.py - I tried a local import to no avail, same issue